discojs-node/README.md

```diff
@@ -0,0 +1,7 @@
+# @epfml/discojs-node
+
+Decentralized & federated privacy-preserving ML training in TypeScript.
+
+This is the node-specific library of the Disco.js project,
+offering some helpers.
+You probably want to also use [`discojs`](../discojs).
```

Comment:
I'm wondering if the local path link won't break once on npm

Reply:
I wonder too, I don't think it'll be an issue, it seems to work for the [@tensorflow/tfjs package](https://www.npmjs.com/package/@tensorflow/tfjs#about-this-repo). it looks like the same file as the one [in the repo](https://github.com/tensorflow/tfjs/blob/68c4de61d530ace87bb0318a4a4b3ca1888dadec/README.md?plain=1#L29).